robottelo/test.py

```diff
@@ -18,8 +18,11 @@
 from datetime import datetime
 from fabric.api import execute, settings
 from robottelo.cli.metatest import MetaCLITest
+from robottelo.cli.org import Org as OrgCli
```

Comment:
Naah, you're right. Let's just `import Org as OrgCli`. We can change this import if a future refactoring occurs.